app/models/order.rb

```diff
@@ -16,4 +16,8 @@ def grand_total
   def total_quantity
     item_orders.sum(:quantity)
   end
+
+  def find_order(merchant)
```

Comment:
@cjkelling Do we have a test for this yet in the spec/models/order_spec.rb file?